crates/telio-traversal/src/connectivity_check/cross_ping_check.rs

```diff
@@ -193,6 +233,7 @@ impl<E: Backoff> CrossPingCheck<E> {
                 poll_timer,
                 ping_pong_handler,
                 exponential_backoff_helper_provider,
+                session_id_candidates: LruCache::new(UPGRADE_TIMEOUT, 512),
```

Comment:
Maybe it would be helpful to have constant for capacity as well? if we have one for timeout value :thinking:. Although this is nitpick TBH.